mne/viz/_3d.py

```diff
@@ -408,8 +408,23 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
     surfs = dict()
     if head:
         subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
-        surfs['head'] = get_head_surf(subject, source=source,
-                                      subjects_dir=subjects_dir)
+        try:
+            surfs['head'] = get_head_surf(subject, source=source,
+                                          subjects_dir=subjects_dir)
```

Comment:
I put this inside a try catch block since ``get_head_surface`` is a public function and modifying it would mean deprecation. Besides, most of the function is just checking for existing files, so doing it here (or through refactoring) would be a bit redundant.